dev/deps/spark-deps-hadoop-3-hive-2.3

```diff
@@ -116,7 +116,6 @@ janino/3.1.9//janino-3.1.9.jar
 javassist/3.25.0-GA//javassist-3.25.0-GA.jar
 javax.jdo/3.2.0-m3//javax.jdo-3.2.0-m3.jar
 javolution/5.5.1//javolution-5.5.1.jar
-jaxb-api/2.2.11//jaxb-api-2.2.11.jar
```

Comment:
Do we need to manually add back this dependency? It disappeared from `hadoop-aliyun`'s dependency chain:

**3.3.4**

```
[INFO] +- org.apache.hadoop:hadoop-cloud-storage:jar:3.3.4:compile
[INFO] |  +- org.apache.hadoop:hadoop-aliyun:jar:3.3.4:compile
[INFO] |  |  \- com.aliyun.oss:aliyun-sdk-oss:jar:3.13.0:compile
[INFO] |  |     +- org.jdom:jdom2:jar:2.0.6:compile
[INFO] |  |     +- org.codehaus.jettison:jettison:jar:1.1:compile
[INFO] |  |     |  \- stax:stax-api:jar:1.0.1:compile
[INFO] |  |     +- com.aliyun:aliyun-java-sdk-core:jar:4.5.10:compile
[INFO] |  |     |  +- javax.xml.bind:jaxb-api:jar:2.2.11:compile
[INFO] |  |     |  +- org.ini4j:ini4j:jar:0.5.4:compile
[INFO] |  |     |  +- io.opentracing:opentracing-api:jar:0.33.0:compile
[INFO] |  |     |  \- io.opentracing:opentracing-util:jar:0.33.0:compile
[INFO] |  |     |     \- io.opentracing:opentracing-noop:jar:0.33.0:compile
[INFO] |  |     +- com.aliyun:aliyun-java-sdk-ram:jar:3.1.0:compile
[INFO] |  |     \- com.aliyun:aliyun-java-sdk-kms:jar:2.11.0:compile
[INFO] |  \- org.apache.hadoop:hadoop-azure-datalake:jar:3.3.4:compile
[INFO] |     \- com.microsoft.azure:azure-data-lake-store-sdk:jar:2.3.9:compile
```

**3.3.5**

```
INFO] +- org.apache.hadoop:hadoop-cloud-storage:jar:3.3.5:compile
[INFO] |  +- org.apache.hadoop:hadoop-annotations:jar:3.3.5:compile
[INFO] |  +- org.apache.hadoop:hadoop-aliyun:jar:3.3.5:compile
[INFO] |  |  +- com.aliyun.oss:aliyun-sdk-oss:jar:3.13.0:compile
[INFO] |  |  |  +- org.jdom:jdom2:jar:2.0.6:compile
[INFO] |  |  |  +- com.aliyun:aliyun-java-sdk-core:jar:4.5.10:compile
[INFO] |  |  |  |  +- org.ini4j:ini4j:jar:0.5.4:compile
[INFO] |  |  |  |  +- io.opentracing:opentracing-api:jar:0.33.0:compile
[INFO] |  |  |  |  \- io.opentracing:opentracing-util:jar:0.33.0:compile
[INFO] |  |  |  |     \- io.opentracing:opentracing-noop:jar:0.33.0:compile
[INFO] |  |  |  +- com.aliyun:aliyun-java-sdk-ram:jar:3.1.0:compile
[INFO] |  |  |  \- com.aliyun:aliyun-java-sdk-kms:jar:2.11.0:compile
[INFO] |  |  \- org.codehaus.jettison:jettison:jar:1.5.3:compile
[INFO] |  \- org.apache.hadoop:hadoop-azure-datalake:jar:3.3.5:compile
[INFO] |     \- com.microsoft.azure:azure-data-lake-store-sdk:jar:2.3.9:compile
```

Reply:
https://github.com/apache/hadoop/commit/72f8c2a4888e2afc1456f3364751afa6c012ed67

exclude `jaxb-api` from  `aliyun-sdk-oss`